### PR TITLE
Name of Stormwind game rule

### PR DESCRIPTION
**Pull request by Grekopithikos**
## Changelog:
- Made a game rule about whether the name of the kingdom of Stormwind can be changed into Azeroth, and its people Azerothians as they were called in the RTS games. It was inspired by vanilla's Name of Byzantium rule from the Roads to Power update.

## Developer changelog:
- Technically the people of Stormwind in the first games were called Azerothiens but whatever. Maybe add this as an extra rule depending on if people want it?

## Tests:
- [ ] There are no errors in `wc` files in `Documents\Paradox Interactive\Crusader Kings III\logs\error.log` except `portrait_decals.cpp:101`
- [ ] The mod takes less than 5.5 GB in the Task Manager (Windows)

# How to test:

### CODE REVIEW

**Comment by robmart** [comment]
Regarding the culture `set_culture_name` is an effect

**Comment by Grekopithikos** [comment]
> Regarding the culture `set_culture_name` is an effect

How about the adjective?

**Comment by helloruhe** [comment]
@Grekopithikos what needs to done for this to be merged

**Comment by Grekopithikos** [comment]
> @Grekopithikos what needs to done for this to be merged

nothing else

File: common/game_rules/000_wc_game_rules.txt [hunk 1]
```diff
@@ -243,4 +243,25 @@ wc_admin_pandaria_gameplay = {
 	wc_admin_pandaria_gameplay_not_administrative = {}
 
 	wc_admin_pandaria_gameplay_always = {}
+}
+
+name_of_stormwind = {
+	
+	categories = {
+		historicity
```
Comment (helloruhe) [inline common/game_rules/000_wc_game_rules.txt]: add warcraft as a category

File: localization/english/wc_game_rules_l_english.yml [hunk 2]
```diff
@@ -93,4 +93,12 @@
 
  rule_wc_admin_pandaria_gameplay: "Extra [administrative|E] [realms|E] — Pandaria"
  setting_wc_admin_pandaria_gameplay_always: "#high Always Administrative#!" setting_wc_admin_pandaria_gameplay_always_desc: "The Empire of Pandaria will always start with an [administrative|E] [government|E] or has a chance to become when formed."
- setting_wc_admin_pandaria_gameplay_not_administrative: "Not Administrative" setting_wc_admin_pandaria_gameplay_not_administrative_desc: "The Empire of Pandaria will not be an [administrative|E] [realm|E] (if controlled by AI)."
\ No newline at end of file
+ setting_wc_admin_pandaria_gameplay_not_administrative: "Not Administrative" setting_wc_admin_pandaria_gameplay_not_administrative_desc: "The Empire of Pandaria will not be an [administrative|E] [realm|E] (if controlled by AI)."
+
+ rule_name_of_stormwind: "#clickable #Bold Warcraft#!#!: Name of Stormwind and its people"
```
Comment (helloruhe) [inline localization/english/wc_game_rules_l_english.yml]: Can simpify, like "Name of Stormwind & Citizens" or smth

**Review by helloruhe — DISMISSED** [review]
just fixed what i said and ur free to merge

**Review by helloruhe — APPROVED** [review]
lgtm

**Pull request merged**